### ISSUE TITLE
Trouble re-installing Electrum on Mac OS X 10.7.5

### ISSUE THREAD

**Issue by abbottjam** [spesmilo/electrum#4890]
Recently after launching Electrum, the bottom panel began showing "Synchronizing..." and my latest transaction didn't show. This went on for sevreal days. 

After doing some research, I decided to install the latest Electrum version, but my Mac cannot open it. The installer just flashes and then nothing happens. 

I tried going back in the version history (on https://download.electrum.org/) to find a compatible one. I tried versions 3.2.2, 3.0, and 2.9.3. At install, every one of them showed the message "Package is damaged. Please eject the disk image."

Finally I tried installing the dev version (https://github.com/spesmilo/electrum#development-version). Cloned the Github directory, cd'ed into it, then went:

> python3 -m pip install .[fast]

I get: 

> /Library/Frameworks/Python.framework/Versions/3.2/Resources/Python.app/Contents/MacOS/Python: No module named pip

In short, not the best of experiences and the problem isn't solved. I'd appreciate info on how to re-install, and most importantly how to do it SECURELY, without exposing any critical data. Thanks!

**Comment by ecdsa** [comment]
your version of Mac OS is very old.
the best way to increase your security is to upgrade it.

**Comment by SomberNight** [comment]
Not really. The currently released binaries require MacOS 10.10 (https://github.com/spesmilo/electrum/issues/3685)
As you can see from that issue, Qt, the GUI framework that we use is the bottleneck.

You could try running from source with an older version of PyQt (maybe 5.8 or 5.9), not sure if it will work.

Why do you want to use that old version of MacOS? It's hard to believe that's your day-to-day machine. Do you use it for offline signing? You could install Linux in a VM.

**Comment by abbottjam** [comment]
That makes it more clear, thanks. 

This machine is from 2008 and yes, it's my day-to-day machine which happens to suit my (limited) needs at the moment. That said, it runs almost all of the programs I've installed through the years without fail.

I'm not using it for offline signing (I probably should). 

> You could install Linux in a VM.

Now that's something I haven't considered — cheers.
